[docs]: update examples + browser config (#777)
browserbase/stagehand: docs/get_started/quickstart.mdx

  </Step>
  <Step title="Create a new project">
    You can use [npx](https://docs.npmjs.com/cli/v8/commands/npx) to create a new project. You should have npx included with `npm`, the default package manager for Node.js.

    <Tabs>
      <Tab title="npm">
        ```bash
        npx create-browser-app
        ```
      </Tab>
      <Tab title="pnpm">
        ```bash
        pnpm create browser-app
        ```
      </Tab>
      <Tab title="yarn">
        ```bash
        yarn create browser-app
        ```
      </Tab>
    </Tabs>
    To use our [Contexts](https://docs.browserbase.com/features/contexts) with Stagehand, run:

    <Tabs>
      <Tab title="npm">
        ```bash
        npx create-browser-app --example persist-context
        ```
      </Tab>
      <Tab title="pnpm">
        ```bash
        pnpm create browser-app --example persist-context
        ```
      </Tab>
      <Tab title="yarn">
        ```bash
        yarn create browser-app --example persist-context
        ```
      </Tab>
    </Tabs>

>     For quick deployment on [Vercel](https://vercel.com/), run:

Reviewer: ```suggestion
    You can easily deploy Stagehand scripts to [Vercel](https://vercel.com/) in one line! Simply run:
```

    <Tabs>
      <Tab title="npm">
        ```bash

>         npx create-browser-app --example deploy-vercel

Reviewer: ```suggestion
        npx create-browser-app --example deploy-vercel
        npx vercel deploy
```

        ```
      </Tab>
      <Tab title="pnpm">
        ```bash
        pnpm create browser-app --example deploy-vercel
        ```
      </Tab>
      <Tab title="yarn">
        ```bash
        yarn create browser-app --example deploy-vercel
        ```
      </Tab>
    </Tabs>

>     To use Computer Use Agents (CUA, Operator), run:

Reviewer: ```suggestion
    We also have an example using CUA agents, where we put OpenAI Operator against Anthropic's Claude Computer Use in a heated game of chess. It's just ten lines of Stagehand code, try it with:
```

    <Tabs>
      <Tab title="npm">
        ```bash

>         npx create-browser-app --example chess

Reviewer 2: should change scaffolding? or this is fine

>         npx create-browser-app --example chess

Reviewer: this is fine

        ```
      </Tab>
      <Tab title="pnpm">
        ```bash
        pnpm create browser-app --example chess
        ```
      </Tab>
      <Tab title="yarn">
        ```bash
        yarn create browser-app --example chess
        ```
      </Tab>
    </Tabs>

>     To use the pre-release version of Stagehand, run:

Reviewer: ```suggestion
    You can also use the version of Stagehand that's on main but not released yet. This will have the bleeding edge features, but they may not be fully prod-ready yet. To test this version, try:
```
